### ISSUE TITLE
Careers Page is slow and theme toggling shows error

### ISSUE THREAD

**Issue by mozi47** [supabase/supabase#17823]
# Bug report

The Careers Page is slow, and takes time to load.

Also when toggling dark and light the on Careers Page, this error appears.  
![image](https://github.com/supabase/supabase/assets/20722868/052581e2-8688-489e-88cb-efc568fa034d)

**Comment by nwaliaez** [comment]
Hey @mozi47 
I tried it worked for me
Steps - 
Open link > https://supabase.com/careers > Scroll to the very bottom > click on theme icon on the bottom right side.
Could you please help if I am missing something ?

**Comment by shareef99** [comment]
I am using Microsoft Edge and the career page is really slow.

**Comment by mozi47** [comment]
On mobile it works fine. Try on desktop. And try on edge and firefox. On chrome sometime it works and some time not.

**Comment by miguelcarvalho13** [comment]
The slowness when scrolling the page up/down also happens to me on a windows laptop with an integrated gpu (both Chrome and Edge). My bet was that the globe component could be the root cause of the issue. So I tried running the careers page locally without the cobe globe and it indeed runs way more smoothly. Another thing I've noticed is that running the careers page with the globe but without the markers also feels ok.

There is an issue in the cobe repository related to that: https://github.com/shuding/cobe/issues/78.

**Comment by spirit-124** [comment]
Hey @mozi47 I don't  know if  I am mistaken at something but it did worked for me.

I tried it By:-
Open link > https://supabase.com/careers > Scroll to the very bottom > click on theme icon on the bottom right side.

**Comment by hichemfantar** [comment]
Can confirm that switching themes breaks the website with a client side error.

**Comment by burggraf** [comment]
I've tested this on both chrome and edge and can't duplicate the error.  Can you please confirm it's still happening to you?

**Comment by mozi47** [comment]
Yes the issue still exist. May be I havent add detail information about the error.

On desktop, on chrome browser
Navigatge to career page 
Scroll down to the footer section
Change the theme and it will show the 404 error.

The page load is also slow.

**Comment by mozi47** [comment]
@nwaliaez @burggraf 

Sorry to record on a mobile phone. for some reason, obs was not working properly,

On Edge:

https://github.com/supabase/supabase/assets/20722868/a5ce29b0-386b-40d4-b416-c8977de6dc9b

On Chrome:

https://github.com/supabase/supabase/assets/20722868/00e5f006-2cd6-4bc3-bbaa-54e0dcd48414


And the career page on all of these browsers is really slow. My PC got stuck as well if you stay on the career page a little longer.

**Comment by hichemfantar** [comment]
In fact, There's a general slowness to the entire website, the careers page is just an extreme example.

**Comment by mozi47** [comment]
Tested, same issue. I tried Firefox as well and the page is extremely slow.

**Comment by encima** [comment]
Unable to reproduce so closing for now until a consistent reproduction can be made

**Comment by hichemfantar** [comment]
The crashing seems to be fixed but there are still some stutters when navigating the website but the performance has definitely improved.

**Comment by hichemfantar** [comment]
The performance degradation has returned and it seems like the globe is the cause.
I removed the canvas element from the dom and the page is smooth again.
i think there's a problem where the globe is being rendered even when it's off screen.

**Comment by supersu-man** [comment]
This issue still exists. The Careers Page lags a lot even when just scrolling. As a work-around I disabled "Hardware Acceleration" on my browser, it now lags a little less, able to scroll to the positions now.
